### PR TITLE
ci(GitHub Actions): add automated update check

### PR DESCRIPTION
**Pull request by hal-shu-sato**
## Description

Check version updates of packages that are released on GitHub.

- Check version update at 05:43 (JST) every day.
- Create a pull request if any updates are available.

## Checklist:

- [ ] ~~Updated the modification date in `mod.xml`.~~

### CODE REVIEW

File: .github/workflows/check-update.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    name: Check Update
+    runs-on: ubuntu-latest
+    env:
+      TZ: 'Asia/Tokyo'
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```
Comment (mitosagi) [inline .github/workflows/check-update.yml:18]: The replaceAll function is supported in node.js 15 and above.
https://stackoverflow.com/a/67877325

**Review by mitosagi — CHANGES_REQUESTED** [review]
I can't tell if the github workflow is correct until I actually run it. Other than that, I've reviewed it.

File: .github/workflows/check-update.yml (hunk 1 above)
Comment (hal-shu-sato) [reply .github/workflows/check-update.yml:18]: Oh, I don't know that.
I'll fix it.

**Comment by hal-shu-sato** [comment]
I fixed it.

**Review by mitosagi — APPROVED** [review]
It''s a useful feature. I would like to see the ability to update mod.xml as a standalone feature as well.

**Comment by hal-shu-sato** [comment]
I think so too!
Thanks for your suggestion.
I'll think of the way.

**Pull request merged**